plugin.json

```diff
@@ -96,10 +96,17 @@
       },
       {
         "key": "DisplaySubscriptionNameInNotifications",
-        "display_name": "Display subscription name in notifications",
+        "display_name": "Display subscription name in notifications:",
         "type": "bool",
         "help_text": "Display subscription name in post when a subscription posts to a channel",
         "default": false
+      },
+      {
+        "key": "EnableWebhookEventLogging",
+        "display_name": "Enable Webhook Event Logging:",
+        "type": "bool",
+        "help_text": "Allow the plugin to log the event from webhook and the log level needs to be set to DEBUG.",
```

Comment:
```suggestion
        "help_text": "Allow the plugin to log the webhook event. The log level needs to be set to DEBUG.",
```

Reply:
If this supports markdown can we surround `DEBUG` with backticks?

```go
    "help_text": "Allow the plugin to log the webhook event. The log level needs to be set to `DEBUG`.",
```

Reply:
@mickmister sure , it looks great!

![image](https://user-images.githubusercontent.com/86650698/142654797-9ce21f38-88c8-49fc-b9ab-c7dd0c9e2200.png)